Dockerfile.ci

```diff
@@ -415,23 +412,23 @@ function install_airflow_dependencies_from_branch_tip() {
     set +x
     common::install_packaging_tools
     set -x
+    echo "${COLOR_BLUE}Uninstalling providers. Dependencies remain${COLOR_RESET}"
     # Uninstall airflow and providers to keep only the dependencies. In the future when
     # planned https://github.com/pypa/pip/issues/11440 is implemented in pip we might be able to use this
     # flag and skip the remove step.
-    ${PACKAGING_TOOL_CMD} freeze | grep apache-airflow-providers | xargs ${PACKAGING_TOOL_CMD} uninstall ${EXTRA_UNINSTALL_FLAGS} 2>/dev/null || true
+    pip freeze | grep apache-airflow-providers | xargs ${PACKAGING_TOOL_CMD} uninstall ${EXTRA_UNINSTALL_FLAGS} || true
```

Comment:
Because these two behave slightly different (different exclusions, different name normalization) - and @Taragolis insisted that in this case the gain (0.5s vs 0.1s) is negligible. Here it is less problematic than in case of constraint generation, but I wanted to keep consistency.